pkg/cmd/server/kubernetes/master/controller/core.go

```diff
@@ -36,31 +35,12 @@ func (c *SchedulerControllerConfig) RunController(ctx kubecontroller.ControllerC
 		ctx.InformerFactory.Extensions().V1beta1().ReplicaSets(),
 		ctx.InformerFactory.Apps().V1beta1().StatefulSets(),
 		ctx.InformerFactory.Core().V1().Services(),
-		c.HardPodAffinitySymmetricWeight,
-	)
-
-	var (
-		config *scheduler.Config
-		err    error
+		recorder,
 	)
-
-	if c.SchedulerPolicy != nil {
```

Comment:
CreateScheduler take care of that. It calls NewFromConfigurator ->Create that reads from policy config file. Now the scheduler also allows to read from ConfigMap. If we want to do that as an option, I could do that too.